File: sakai-ng-master/src/app/Models/EmpleadoViewModel.ts
export class Empleado{
    muni_Municipio?:String;
    esta_EstadoCivil?:String;
    carg_Cargo?:String;
    empl_Id?:String;
    empl_Nombre?:String;
    empl_Apellido?:String;
    empl_Sexo?:String;    
    empl_FechaNac?:String;
    empl_Correo:string;

    empl_DNI?:String;
    Muni_Municipio:string;

}

export class dropEmpleado{
    value?:String;
    text?:String;
}

export class EmpleadoEnviar {
    


    Usua_Id?: Number;
    Empl_DNI?:String;
    Empl_Id:string;
    Empl_Nombre: string;
    Empl_Apellido: string;
    Empl_FechaNac: string;
    Empl_Sexo: string;
    Muni_Codigo: string;
    Esta_Id: string;
    Carg_Id: string;
    Sucu_Id: string;
    Empl_Correo:string;


}

export class Fill {
    empl_Id: string;
    empl_Nombre: string;
    empl_Apellido?:String;
    empl_FechaNac?:String;
    empl_Sexo?:String;
    muni_Codigo?:String;
    esta_Id?:String;
    carg_Id :string;
    carg_Cargo?:string;
    esta_EstadoCivil?:String;
    muni_Municipio?:String;
    empl_Correo:string;
    empl_DNI?:String;
    depa_Codigo?:String;
    depa_Departamento?:String;
    usuarioCreacion: string;
    usuarioModificacion: string;
    fechaCreacion : string;
    fechaModificacion : string;
}

File: API Buena/SistemaEsmeralda/SistemaEsmeralda.Common/Models/EmpleadoViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaEsmeralda.Common.Models
{
    public class EmpleadoViewModel
    {
        public int Empl_Id { get; set; }
        public string Empl_Nombre { get; set; }
        public string Empl_Apellido { get; set; }
        public string Empl_Sexo { get; set; }
        public DateTime Empl_FechaNac { get; set; }
        public string Muni_Codigo { get; set; }
        public int Esta_Id { get; set; }
        public int Carg_Id { get; set; }
        public int Sucu_Id { get; set; }
        public int? Empl_UsuarioCreacion { get; set; }
        public DateTime? Empl_FechaCreacion { get; set; }
        public int? Empl_UsuarioModificacion { get; set; }
        public DateTime? Empl_FechaModificacion { get; set; }
        public bool? Empl_Estado { get; set; }

        [NotMapped]
        public string UsuarioCreacion { get; set; }

        [NotMapped]
        public string UsuarioModificacion { get; set; }

        [NotMapped]
        public string Muni_Municipio { get; set; }
        [NotMapped]
        public string Esta_EstadoCivil { get; set; }

        [NotMapped]
        public string Empl_Correo { get; set; }


        
    }
}
